x-pack/plugins/cases/public/common/shared_imports.ts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Do we need this file for cases?

Reply:
yes! we import the form from `es_ui_shared`